src/collections/blog/2022-02-25-layer5-collab-with-openforce/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Layer5 collaboration with OpenForce 2022"
+date: 2022-02-25 
+author: Shebeul Inyang
+thumbnail: 
+category: "Community"
+tags:
+ - Community
+ - Events
+ - OpenSource
+
+featured: true
+published: true
+---
+
+
+<h2 id="collaboration">Layer5 collaboration with OpenForce 2022</h2>
+<p>
+Great News!🎉🎉   
+OpenForce2022 by CodeFlow is here! And Layer5 is joining forces with other open source organizations to make it an unforgettable experience. 
+OpenForce 2022 is a place where contributors can meet mentors from open source organizations and contribute to projects under guidance, network with project maintainers, and “bring the open source culture” with great enthusiasm.
+</p>
+<h2 id="tasks">What Layer5 will do?</h2>
+<p>
+During the program, Layer5 as an open source organization will provide OpenForce specific issues which participants can resolve in the Contribution Period. In addition, there will be a panel of mentors assigned to assist contributors overcome any challenges they might face.
+Beyond the direct support offered by Layer5 mentors, program participants will have an amazing networking opportunity to connect with the broader set of community members at Layer5. This event will kick off virtually from March 3rd to March 28th 2022. 
+</p>
+<h2 id="codeflow">CodeFlow</h2>
+<p>
+The program is organized by CodeFlow, an organization whose main motive is to empower the student community by organizing webinars, hackathons, and open source events, providing  them with the best learning environment and helping them gain the principal knowledge about the fast-paced, open source ecosystem. What a great learning opportunity this would prove to be as ‘open source meets its contributor force’ at OpenForce 2022! Come be a part of this amazing initiative.

```

Comment:
```suggestion
The program is organized by CodeFlow, an organization whose main motive is to empower the student community by organizing webinars, hackathons, and open source events, providing them with the best learning environment and helping them gain the principal knowledge about the fast-paced, open source ecosystem. What a great learning opportunity this would prove to be as ‘open source meets its contributor force’ at OpenForce 2022! Come be a part of this amazing initiative.

```